tests/unit/test_report_show_e2e.py

```
@@ -0,0 +1,247 @@
+
+		captured = {}
+
+		def capture_send(report_self):
+			captured['results'] = report_self.data['results']
+
+		args = ['r', 'show', '-w', WORKSPACE, '--driver', 'local']
+		if query_expr:
+			args += ['-q', query_expr]
+
+		with mock.patch('secator.query.json.CONFIG') as mock_cfg, \
+				mock.patch('secator.report.Report.send', capture_send):
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)